packages/terra-functional-testing/tests/jest/commands/expect/toMatchReference.test.js

```diff
@@ -210,10 +210,10 @@ describe('toMatchReference', () => {
     expect(result.pass).toBe(true);
   });
 
-  it('should not pass if not within mismatch tolerance, buildBranch is pullRequest, and useRemoteReferenceScreenshots is false', () => {
+  it('should not pass if not within mismatch tolerance, buildBranch matchs pullRequest, and useRemoteReferenceScreenshots is false', () => {
```

Comment:
"matches". Not a big deal. Probably not worth a rebuild.

Reply:
Fixed here since i had type and had to rebuild anyway
https://github.com/cerner/terra-toolkit/pull/737/commits/1a7a6078b6d9d8aa58d05518a8929dfa8dcf5f1b